lib/coverband/utils/coverband_results.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+####
+# A way to access the various coverage data breakdowns
+####
+module Coverband
+  module Utils
+    class CoverbandResults
+      attr_accessor :type, :results
+
+      def initialize(report)
+        self.results = {
```

Comment:
Minor suggestion.. holds up if we had more types i think.
```
self.results = Coverband::Types.each_with_object({}) do |type, hash|
  hash[type] = Coverband::Utils::Result.new(report[type])
end
```

Reply:
good call

Reply:
I will update and do another pass on tests tonight

Reply:
ok resolved